tests/losses/test_giou.py

```diff
@@ -0,0 +1,24 @@
+"""
+Test Generalized Intersection over Union
+"""
+
+from unittest import TestCase
+
+import torch
+
+from pl_bolts.losses.giou import giou_loss
+
+
+class TestGIoULoss(TestCase):
```

Comment:
can we rather write it as simple `test_complete_overlap` with parameterized inputs and expected results?
it would be easier to debug later as we would si what exact configurations fails

Reply:
I'm unfamiliar with `unittest` (I'm used to `pytest`) so I didn't realize you can do that in `unittest` as well!

Reply:
I realized that `pytest` is being used in conjunction with `unittest`, so I just used `pytest` to parameterize the tests.

Reply:
happy to help if you want, just ping me on slack :]